### PR TITLE
Add missing help text for CLI commands 

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
Add the CLI help text commands for commands using the docstring, whenever a command using a docstring that have args as a help. it shows the args in an unformatted way.

While there is a solution to rewrap this for click to show the args each in a line. this doesn't work with our docstring checking library.

image

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Review by fa9r — APPROVED** [review]
LGTM

File: src/zenml/cli/secret.py [hunk 1]
```diff
@@ -92,56 +96,10 @@ def register_secret(
 
     Use this command to store sensitive information into a ZenML secret. The
     secret data consists of key-value pairs that can be configured interactively
-    (if the `--interactive` option is set) or via command line arguments.
+    (if the `--interactive` option is set) or via command-line arguments.
     If a schema is indicated, the secret key-value pairs will be validated
     against the schema.
 
-    When passed as command line arguments, the secret field values may also be
```
Comment (strickvl) [inline src/zenml/cli/secret.py:99]: Does this text just get deleted? Are we moving it somewhere else?

Comment (safoinme) [reply src/zenml/cli/secret.py:99]: Ah, that was deleted with the old way cause it didn't make sense to show it in CLI. I return it back until figure out where that should be written.

**Review by strickvl — APPROVED** [review]
Yeah once the linting issues with darglint etc are fixed, this looks good to me now!

**Pull request merged**